src/sidebar/components/Annotation/AnnotationThumbnail.tsx

```diff
@@ -52,23 +55,54 @@ function AnnotationThumbnail({
     altText = 'Thumbnail';
   }
 
+  const scaledWidth = thumbnail ? thumbnail.width / devicePixelRatio : 0;
+  const scaledHeight = thumbnail ? thumbnail.height / devicePixelRatio : 0;
+
   return (
     <div
-      className="flex flex-row justify-center"
+      className="flex flex-row justify-center relative"
       data-testid="thumbnail-container"
     >
       {thumbnail && (
         <img
           src={thumbnail.url}
           alt={altText}
-          title={altText}
           className="border rounded-md"
           style={{
-            width: `${thumbnail.width / devicePixelRatio}px`,
-            height: `${thumbnail.height / devicePixelRatio}px`,
+            width: `${scaledWidth}px`,
+            height: `${scaledHeight}px`,
           }}
         />
       )}
+      {thumbnail && description && (
+        <>
+          <IconButton
+            classes="absolute top-0 text-[white] text-[16px]"
+            variant="custom"
+            icon={InfoIcon}
+            title="Image description"
+            onClick={() => setPopoverOpen(true)}
+            expanded={popoverOpen}
+            elementRef={descriptionButtonRef}
+            style={{
+              // Position info button towards top-right corner of thumbnail.
+              left: `calc(50% + ${scaledWidth}px / 2 - 32px)`,
+              // Add a drop shadow to make the white icon visible on thumbnails
+              // that have a light background.
+              filter: 'drop-shadow(grey 1px 1px 1px)',
```

Comment:
I reckon tailwind's `drop-shadow-sm` should be pretty similar to this https://v3.tailwindcss.com/docs/drop-shadow